### PR TITLE
Feat/ log branch and commit ver

### PR DESCRIPTION
**Pull request by y0sher**
looks like this now - 
```
2023-11-18T20:14:37.699299Z	[35mINFO[0m	starting ssv-node { "version":  "v1.0.0-rc.2-44-g2409736e-stage-2409736e1e1b22e1c6e882821075eb38cc076dde" }
```

### CODE REVIEW

File: cli/bootnode/boot_node.go [hunk 1]
```diff
@@ -41,11 +43,15 @@ var StartBootNodeCmd = &cobra.Command{
 				MaxBackups: cfg.LogFileBackups,
 			},
 		)
+
 		if err != nil {
 			log.Fatal(err)
 		}
 
 		logger := zap.L()
+		log.Printf("starting SSV node (version %s)", commons.GetBuildData())
```
Comment (nkryuchkov) [inline cli/bootnode/boot_node.go:52]: Is there any reason not to use `logger` here? Why do we have two similar logs?

File: cli/bootnode/boot_node.go [hunk 2]
```diff
@@ -41,11 +43,15 @@ var StartBootNodeCmd = &cobra.Command{
...
+
+		logger.Info(fmt.Sprintf("starting SSV %v (version %s)", cmd.Short, commons.GetBuildData()))
```
Comment (nkryuchkov) [inline cli/bootnode/boot_node.go:54]: The `AppName` is `SSV-Node`, so this seems to be printed as `starting SSV SSV-Node`, should we remove the "SSV" duplication?

File: cli/bootnode/boot_node.go (hunk 1 above)
Comment (y0sher) [reply cli/bootnode/boot_node.go]: you are right. changed.

File: cli/bootnode/boot_node.go (hunk 2 above)
Comment (y0sher) [reply cli/bootnode/boot_node.go]: agree.

Comment (Roy-blox) [reply cli/bootnode/boot_node.go]: @y0sher  Take a look at this: https://blox-ensemble.monday.com/boards/4755515647/pulses/5536501995
It seems commons.GetBuildData returns "SSV node:' ' " with no data in it.
Regardless of the bug I believe a quick validation to make sure that the data is not empty is in place here

**Review by andrew-blox — APPROVED** [review]

**Pull request merged**